src/sidebar/tabcenter.js

```diff
@@ -69,12 +74,24 @@ TabCenter.prototype = {
       document.body.classList.remove("dark-theme");
     }
   },
+  set _themeIntegration(enabled) {
+    if (!enabled) {
+      this._applyTheme({});
```

Comment:
Please create a helper function `_resetTheme` that hides this implementation detail.